json-body-transformer/src/test/java/com/ninecookies/wiremock/extensions/JsonBodyTransformerTest.java

```diff
@@ -309,11 +333,21 @@ public void doesNotApplyGlobally() throws Exception {
 		wireMockServer.verify(postRequestedFor(urlEqualTo(REQUEST_URL)));
 	}
 
-	@Test
-	public void injectRandom() {
+	@DataProvider
+	private Object[][] randomFormats() {
+		return new Object[][] {
+				{ "$(!Random)" },
+				{ "$(!Random.1)" },
+				{ "$(!Random.abc)" },
+				{ "$(!Random.ABC)" },
+		};
+	}
+
+	@Test(dataProvider = "randomFormats")
+	public void injectRandom(String format) {
 		wireMockServer.stubFor(post(urlEqualTo(REQUEST_URL))
 				.willReturn(aResponse().withStatus(200).withHeader("content-type", CONTENT_TYPE)
-						.withBody("{\"randomNumber\":\"$(!Random)\", \"got\":\"it\"}")
+						.withBody("{\"randomNumber\":\"" + format + "\", \"got\":\"it\"}")
```

Comment:
Sorry for mixing up `$(!Random)` and `$(!UUID)` keywords, but in fact both are handled in the same way. Suffix only exists to get different values generated for different properties with the ability to be reused...